source/elements/oneMKL/source/domains/spblas/operations/spmm.rst

```diff
@@ -297,8 +297,11 @@ spmm
 
    alpha
       Host or USM pointer representing :math:`\alpha`. The USM allocation can be
-      on the host or device. Must be a host pointer if SYCL buffers are used.
-      Must be of the same type than the handles' data type.
+      on the host or device. The requirements are:
+
+      * Must use the same kind of memory than ``beta``.
```

Comment:
```suggestion
      * Must use the same kind of memory as ``beta``.
```

Reply:
I've replaced all the occurrences in https://github.com/uxlfoundation/oneAPI-spec/pull/556/commits/ea48f07431c18d6aeaee3716bcf81826e73b02de